src/interfaces.ts

```diff
@@ -99,6 +99,7 @@ export type CurriedEffectRun = (
 )
 
 export interface DeferredEffects {
+  // Block number
```

Comment:
Will this show up in the docs or should it be `/** Block number */`?

Reply:
This won't show up in docs, but I felt like it would be silly to document that one line and nothing else. We should talk if there needs to be a separate PR where all these interfaces are documented. (It's possible that it's unnecessary, as their consuming structures could make them self-explanatory.)